packages/@aws-cdk/aws-eks/lib/helm-chart.ts

```diff
@@ -93,13 +93,14 @@ export class HelmChart extends Construct {
     new CustomResource(this, 'Resource', {
       serviceToken: provider.serviceToken,
       resourceType: HelmChart.RESOURCE_TYPE,
+      encodeValues: true,
       properties: {
         ClusterName: props.cluster.clusterName,
         RoleArn: props.cluster._kubectlCreationRole.roleArn,
         Release: props.release ?? this.node.uniqueId.slice(-53).toLowerCase(), // Helm has a 53 character limit for the name
         Chart: props.chart,
         Version: props.version,
-        Wait: props.wait ?? false,
+        Wait: props.wait || undefined,
```

Comment:
Add a comment explaining why this needs to be “undefined” to represent false

Reply:
@eladb , do you want me to create a separate PR just to address this bug with the `wait` parameters in the helm charts?